src/main/resources/anniv5Resources/localization/eng/bladestormpack/Relicstrings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "${ModID}:FlyingDutchman": {
+    "NAME": "Flying Dutchman",
+    "FLAVOR": "You'd swear it glows and hovers when the weather's real bad.",
+    "DESCRIPTIONS": [
```

Comment:
Looking at the wording for this relic, it would be a bit easier to parse if the wording was closer to Akabeko or Necronomicon. E.g. `The second time you play an Attack each turn, gain 2 Windrush and 3 Precision.`

Also, think you can get rid of the pluralization logic, since the number of attacks to trigger this is fixed.